CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Add reviewer's page to catalog. [#756](https://github.com/rokwire/rokwire-building-blocks-api/issues/756)
```

Comment:
I see that #755 is also linked in this PR. I think the code changes for that is not in  this PR, right?

Reply:
unlinked